packages/mui-material/src/Slider/Slider.js

```diff
@@ -523,55 +593,197 @@ const Slider = React.forwardRef(function Slider(inputProps, ref) {
   const markLabelSlotProps = slotProps?.markLabel ?? componentsProps.markLabel;
   const inputSlotProps = slotProps?.input ?? componentsProps.input;
 
+  const rootProps = useSlotProps({
+    elementType: RootSlot,
+    getSlotProps: getRootProps,
+    externalSlotProps: rootSlotProps,
+    externalForwardedProps: other,
+    additionalProps: {
+      ...(shouldSpreadAdditionalProps(RootSlot) && {
+        as: component,
+      }),
+    },
+    ownerState: {
+      ...ownerState,
+      ...rootSlotProps?.ownerState,
```

Comment:
I wonder what could be the use case for providing `ownerState` in `slotProps.*`. I can't think of any scenarios that would require overriding the state, so if you have anything in mind, please share.

Reply:
I don't have anything in mind or a use case in mind. I had wondered the same while adding this. I later thought that I didn't want to introduce any breaking changes, so let's keep it.